core/src/main/java/com/example/core/components/VerticalList.kt

```diff
@@ -1,16 +1,35 @@
 package com.example.core.components
 
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
 
 @Composable
 fun VerticalList(
     size: Int,
     item: @Composable (Int) -> Unit
 ) {
-    LazyColumn {
-        items(size) { index ->
-            item(index)
+    LazyColumn(
+        modifier = Modifier.fillMaxSize()
+    ) {
+        items(size + 1) { index ->
+            if (index == size)
+                Spacer(modifier = Modifier) else
+                Box(
+                    modifier = Modifier.padding(
+                        10.dp,
```

Comment:
Do not hard-code the value like this; instead, pass this value from the outside and assign a default to it.